Integrations/integration-AzureCompute.yml

```diff
@@ -0,0 +1,1085 @@
+      description: Admin Username to be used when creating the VM
+      isArray: false
+      name: admin_username
+      required: false
+      secret: false
+    - default: false
+      defaultValue: Passw0rd@123
+      description: Admin Password to be used when creating the VM
+      isArray: false
+      name: admin_password
+      required: false
+      secret: false
+    description: Create a virtual machine instance with the specified OS image
+    execution: true
+    name: azure-vm-create-instance
+    outputs:
+    - contextPath: Azure.Compute.Name
+      description: Name of the created VM instance
+      type: string
+    - contextPath: Azure.Compute.ResourceGroup
+      description: Resource group the VM resides in
+      type: string
+    - contextPath: Azure.Compute.ID
+      description: ID of the VM
+      type: string
+    - contextPath: Azure.Compute.Size
+      description: Size of the deployed VM in gigabytes
+      type: number
+    - contextPath: Azure.Compute.OS
+      description: OS running in the specified VM
+      type: string
+    - contextPath: Azure.Compute.ProvisioningState
+      description: Provisioning state of the deployed VM
+      type: string
+    - contextPath: Azure.Compute.Location
+      description: Region in which the VM is hosted
+      type: string
+  - description: List all Resource Groups belonging to your Azure subscription
+    execution: false
+    name: azure-list-resource-groups
+    outputs:
+    - contextPath: Azure.ResourceGroup.Name
+      description: Name of the Resource Group
+      type: string
+    - contextPath: Azure.ResourceGroup.ID
+      description: ID of the Resource Group
+      type: string
+    - contextPath: Azure.ResourceGroup.Location
+      description: Location of the Resource Group
+      type: string
+    - contextPath: Azure.ResourceGroup.ProvisioningState
+      description: Provisioning State of the Resource Group
+      type: string
+  - arguments:
+    - default: false
+      description: Resource Group to which the virtual machine belongs
+      isArray: false
+      name: resource_group
+      required: true
+      secret: false
+    - default: false
+      description: Name of the virtual machine to delete
+      isArray: false
+      name: virtual_machine_name
+      required: false
+      secret: false
+    description: Delete a specified Virtual Machine
+    execution: false
+    name: azure-vm-delete-instance
+  isfetch: false
+  runonce: false
+  script: |-
+    '''IMPORTS'''
+
+    import requests
+
+    '''GLOBAL VARS'''
+
+    PARAMS = demisto.params()
+    USE_SSL = not demisto.params().get('unsecure')
+    CLIENT_ID = PARAMS.get('client_id')
+    CLIENT_SECRET = PARAMS.get('client_secret')
+    SUBSCRIPTION_ID = PARAMS.get('subscription_id')
+    TENANT_ID = PARAMS.get('tenant_id')
+    SERVER = 'https://management.azure.com'
+    BASE_URL = SERVER + "/subscriptions/" + SUBSCRIPTION_ID + "/resourceGroups/"
+    API_VERSION = "2018-06-01"
```

Comment:
I now changed it to be a parameter and not hard-coded, with the default of "2018-06-01" if the user doesn't specify otherwise in the configuration. I chose it because I believe it's the latest version of the API available.